windows_prepare_python.bat

```diff
@@ -0,0 +1,58 @@
+::
+:: Copyright (c) 2024 Intel Corporation
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::      http:::www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+::
+setlocal EnableExtensions EnableDelayedExpansion
+@echo off
+
+set "embeddable_python_url=https://www.python.org/ftp/python/3.9.13/python-3.9.13-embed-amd64.zip"
+
+:: Download and unpack everything 
+if exist C:\opt\python-3.9.13-embed-amd64 (
+    echo Existing Python installation found. Exiting script.
+    exit 0
+) else ( 
+    md C:\opt\python-3.9.13-embed-amd64 
+)
+if !errorlevel! neq 0 exit /b !errorlevel!
+
+if exist C:\opt\python-3.9.13-embed-amd64.zip (
+    echo Python zip already downloaded. Will unpack existing file.
+) else (
+    curl %embeddable_python_url% -o C:\opt\python-3.9.13-embed-amd64.zip
+)
+if !errorlevel! neq 0 exit /b !errorlevel!
+
+tar -xf C:\opt\python-3.9.13-embed-amd64.zip -C C:\opt\python-3.9.13-embed-amd64
```

Comment:
Make python-3.9.13-embed-amd64 an env variable and use it everywhere.

Reply:
Make -C target a parameter to the script (C:\opt\%target_location%)

Reply:
I'm not convinced about the second comment. If we hardcode specific python version I think we can hardcode extraction destination to the same name as the original zip file.